components/brave_rewards/resources/android_page/reducers/wallet_reducer.ts

```diff
@@ -72,17 +64,27 @@ const walletReducer: Reducer<Rewards.State | undefined> = (state: Rewards.State,
       break
     }
     case types.GET_WALLLET_PASSPHRASE:
-      chrome.send('brave_rewards.getWalletPassphrase', [])
+      chrome.send('brave_rewards.getWalletPassphrase')
       break
     case types.ON_WALLLET_PASSPHRASE:
       const value = action.payload.pass
       if (value && value.length > 0) {
         state = { ...state }
+        let ui = state.ui
         state.recoveryKey = value
+        ui.paymentIdCheck = true
+
+        state = {
+          ...state,
+          ui
+        }
       }
       break
-    case types.RECOVER_WALLET:
-      if (!action.payload.key || action.payload.key.length === 0) {
+    case types.RECOVER_WALLET: {
```

Comment:
not sure why we have recover wallet as there is not recover wallet on andoird, but I see that it was there before already

Reply:
Removed.